tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -282,12 +283,13 @@ def test_unfurl_metric_alerts_chart_transaction(self, mock_generate_chart):
         assert chart_data["incidents"][0]["id"] == str(incident.id)
 
     @patch("sentry.incidents.charts.generate_chart", return_value="chart-url")
+    @freeze_time("2022-02-21")
```

Comment:
Actually looks like we already freeze time on the base class. I didn't realise the test fail was a real failure, you can probably just remove this.